treeherder/perf/perf_sheriff_bot.py

```diff
@@ -4,25 +4,119 @@
 from django.conf import settings
 from taskcluster.helper import TaskclusterConfig
 
-from treeherder.perf.alerts import BackfillReport
+from treeherder.perf.alerts import BackfillRecord, BackfillReport, BackfillReportMaintainer
+from treeherder.perf.backfill_tool import BackfillTool
+from treeherder.perf.secretary_tool import SecretaryTool
 
 CLIENT_ID = settings.PERF_SHERIFF_BOT_CLIENT_ID
 ACCESS_TOKEN = settings.PERF_SHERIFF_BOT_ACCESS_TOKEN
 
+'''
+TODO:
+* guard by feature flag
+* READY_FOR_PROCESSING -> BACKFILLED
+* [fails immediately] READY_FOR_PROCESSING -> FAILED
```

Comment:
I'm not sure I understand this part. Will there be records that go from READY_FOR_PROCESSING directly to FAILED?

Reply:
Yes, those that fail immediately.